ppx/ppx_mysql.ml

```diff
@@ -66,23 +72,31 @@ let parse_query =
               { query = Buffer.contents buf
               ; in_params = List.rev acc_in
               ; out_params = List.rev acc_out }
-        | Some _ -> Error `Unterminated_string
+        | Some _ ->
+            Error `Unterminated_string
       else
         let this = query.[i] in
         match string_delim with
-        | _ when Char.(this = '\\') ->
+        | _
+          when Char.(this = '\\') ->
```

Comment:
Would prefer `when` cases to be on the same line (assuming it fits). But this is OK

Reply:
I would like a `break-guards = fit` setting, agreed. Maybe I should file a feature request?